flocker/control/_persistence.py

```diff
@@ -0,0 +1,86 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Persistence of cluster configuration.
+"""
+
+from cPickle import dumps, loads
```

Comment:
Probably no benefit to using cPickle over pickle.  Probably only obscure downsides to using cPickle (bad profiler interaction, bad exception reporting, slightly greater chance of memory corruption, etc).  So not a big deal, but it's easy to switch. :smile: